File: app/src/main/java/com/example/modulo_5/MainActivity.java
package com.example.modulo_5;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;
import android.renderscript.ScriptGroup;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

import com.example.modulo_5.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {


    private ActivityMainBinding mBinding;
    private EditText texto1;
    private Button boton;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mBinding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(mBinding.getRoot());


        texto1 = mBinding.editTxt;
        boton = mBinding.button;
        boton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                mostrarFragmento();
            }
        });
    }
    private void  mostrarFragmento(){
    //crear instania del fragmento
        BlankFragment fragment = BlankFragment.newInstance(texto1.getText().toString(),"");
        //obtener el administrador de los fragmentos
        FragmentManager fragmentManager = getSupportFragmentManager();
        //iniciar transaccion del fragmento
        FragmentTransaction transaction = fragmentManager.beginTransaction();
        //reemplazar el contenido del contenedor con el fragmento
        transaction.replace(mBinding.container.getId(), fragment);
        //opcional : añadir la transacion la pila de retroeso
        transaction.addToBackStack(null);
        transaction.commit();

    }
}

File: app/src/main/java/com/example/recyclewviewkotlin/MainActivity.kt
package com.example.recyclewviewkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.example.recyclewviewkotlin.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding
    private var listadatos: MutableList<Adaptador> = mutableListOf()
    private lateinit var recycler: RecyclerView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        listadatos.add(Adaptador(("item 1")))
        listadatos.add(Adaptador(("item 2")))
        listadatos.add(Adaptador(("item 3")))
        listadatos.add(Adaptador(("item 4")))
        listadatos.add(Adaptador(("item 5")))
        listadatos.add(Adaptador(("item 6")))
        listadatos.add(Adaptador(("item 7")))
        listadatos.add(Adaptador(("item 8")))
        listadatos.add(Adaptador(("item 9")))
        listadatos.add(Adaptador(("item 10")))
        listadatos.add(Adaptador(("item 11")))
        listadatos.add(Adaptador(("item 12")))
        listadatos.add(Adaptador(("item 13")))
        listadatos.add(Adaptador(("item 14")))
        listadatos.add(Adaptador(("item 15")))

        establecerAdaptador()


    }
    private fun establecerAdaptador(){
        recycler = binding.lista
        recycler.layoutManager = LinearLayoutManager(this)
        recycler.adapter = Reciclador(this,listadatos)
    }
}